code/modules/mob/living/carbon/human/species/shadekin/shadekin_abilities.dm

```diff
@@ -95,7 +95,7 @@
 			var/list/potentials = living_mobs(0)
 			if(potentials.len)
 				var/mob/living/target = pick(potentials)
-				if(istype(target) && vore_selected)
+				if(target.devourable && target.can_be_drop_prey && istype(target) && vore_selected)
```

Comment:
istype(targer) should probably come before the checks, just in case. Applies to other lines too.